### DISCUSSION TITLE
Feature Request: Add "g+s" Keyboard Shortcut for Navigating to Starred Repositories

### DISCUSSION THREAD

**Discussion by ghost** [community/community#118903] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

As an avid user of GitHub, I often find myself navigating to my starred repositories to revisit projects of interest. Currently, accessing the list of starred repositories requires several mouse clicks, which can disrupt workflow efficiency. Therefore, I propose the implementation of a new site-wide keyboard shortcut, "g+s", to directly navigate users to their starred repositories page.

**Comment by SupunJayaweera** [comment]
Hello there,

That sounds like a great idea! Having a "g+s" keyboard shortcut for navigating to starred repositories could indeed enhance the user experience and make navigation more efficient. 

As GitHub is always looking to improve and streamline its user experience, I believe your suggestion would be a valuable addition. 

To get this feature request to the right people, I recommend opening an issue in the official GitHub public roadmap repository. This repository is where GitHub tracks upcoming features and updates, and it's the best place to suggest new ideas.

When you open the issue, be sure to provide as much detail as possible about your proposed feature. Explain why you think it would be beneficial, how it could work, and any potential challenges you foresee.

Happy coding!
